src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -103,7 +133,12 @@ export default {
     async stopRecording() {
       if (!this.recorder) return;
       await new Promise(resolve => {
-        this.recorder.stopRecording(resolve);
+        if (this.stream.active) {
```

Comment:
what is the reason to check stream with active state?

Reply:
this is for fix flow with ff which I describe above
Chrome require page reload when you change permissions, but Firefox (maybe something another browser) don't require it, you can change permissions in real time
when you allow camera, start record and then immediately revoke permissions, old logic will think that we successfully record selfie and will provide us the `continue` button to the next step
this is incorrect

Reply:
when stream is not null (we drop it if recording ends correctly) and is not active, it means, correct flow was dropped by revoking permissions or camera was disconnected
so we should catch it and give to user opportunity allow camera again, or show error

Reply:
please describe the case, how to reproduce it?